GUI/SearchTaskWindow.xaml.cs

```diff
@@ -394,7 +402,32 @@ private void ApmdExpander_Collapsed(object sender, RoutedEventArgs e)
             dataContextForSearchTaskWindow.SearchModeExpanderTitle = "Some search properties...";
         }
 
+        private static void PreviewIfInt(object sender, TextCompositionEventArgs e)
```

Comment:
I'm spending too much time trying to be fancy, so just going to make a check at save time.